Source/JavaScriptCore/inspector/protocol/CSS.json

```diff
@@ -253,7 +253,9 @@
             "description": "A representation of WebCore::Font. Conceptually this is backed by either a font file on disk or from the network.",
             "properties": [
                 { "name": "displayName", "type": "string", "description": "The display name defined by the font." },
-                { "name": "variationAxes", "type": "array", "items": { "$ref": "FontVariationAxis" }, "description": "The variation axes defined by the font." }
+                { "name": "variationAxes", "type": "array", "items": { "$ref": "FontVariationAxis" }, "description": "The variation axes defined by the font." },
+                { "name": "syntheticBold", "type": "boolean", "optional": true, "description": "Whether the font has synthesized its boldness or not." },
+                { "name": "syntheticOblique", "type": "boolean", "optional": true, "description": "Whether the font has synthesized its obliqueness or not" }
```

Comment:
NIT: Maybe `synthesizedBold` and `synthesizedOblique` to make it sound more like a verb instead of a noun (i.e. "what is the synthetic bold" vs "is it a synthesized bold")?